tracing/src/macros.rs

```diff
@@ -2784,7 +2886,9 @@ macro_rules! level_enabled {
 
 #[doc(hidden)]
 #[macro_export]
-macro_rules! valueset {
+// `valueset_arr` is only suitable as a `match` scrutinee location
+// because the `Value`s assume a very transient temporary lifetime
+macro_rules! valueset_arr {
```

Comment:
why the rename? I would maybe consider keeping it the *same* name, especially since this is a hidden, semver-exempt API.

Reply:
Sure. Rename is reverted.